samples/java/hadoop/sources/LinearRegressionNormEq.java

```diff
@@ -51,8 +51,14 @@
 /* Implement Tool to be able to pass -libjars on start */
 public class LinearRegressionNormEq extends Configured implements Tool {
 
-    public static void main(String[] args) throws Exception {
-        int res = ToolRunner.run(new Configuration(), new LinearRegressionNormEq(), args);
+    public static void main(String[] args) {
+        int res = -1;
+        try {
+            res = ToolRunner.run(new Configuration(), new LinearRegressionNormEq(), args);
+        } catch (Exception e) {
+            System.out.println("Error");
+            e.printStackTrace();
```

Comment:
Let's print more meaningful diagnostic. I think that stack trace is not enough.
            System.out.println("Exception caught: " + e);
            System.out.println("Exception message: " + e.getMessage());
            System.out.println("Exception cause: " + e.getCause());
            e.printStackTrace();

Same comment to the rest of the samples.